api/controllers/artists.py

```diff
@@ -3,8 +3,15 @@
 from sqlalchemy.orm import Session
```

Comment:
(3/4) ...and then maybe we could also move sqlalchemy imports down, to keep all `api` imports together?
```suggestion
```